app/Http/Controllers/User/UserController.php

```diff
@@ -54,28 +54,39 @@ public function myProfile()
                 'followers',
                 'followings',
             ]);
+        $id = Auth::id();
 
-        return $this->getUserInfo($user);
+        return $this->getUserInfo($user, $id);
     }
 
-    public function getBooks($status, $id)
+    public function postMyProfile($id)
     {
-        $data = [
-            'user_id' => $id,
-            'type' => $status,
+        $selects = [
+            'id',
         ];
         $with = [
-            'book',
+            'ownerBooks',
         ];
-        $books = $this->bookUser->getData($data, $with)->pluck('book');
-        $books = $books->chunk(config('view.paginate.book_profile'));
+        $user = $this->user->find($id, $with, $selects);
+        $books = $user->ownerBooks()->where('user_id', $id)->paginate(config('view.limit.related_book'));
+        
+        return view('layout.section.sharing_books', compact('books'));        
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/UserController.php</br>**Line**: 73</br>Whitespace found at end of line